CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Modify partitions label, define entire resinOS space used to 700MiB and other minor changes [Andrei]
```

Comment:
It is still work in progress deciding the final number. For now I submitted this for review so we can speed up the process. Hopefully on Monday we will have the final decision. I personally don't think 600 is enough. At this point it leaves us with 80M free space (which doesn't include docker update, debug tools, provisioner etc).